spec/features/events_spec.rb

```diff
@@ -52,4 +52,33 @@
     end
     page.should have_content 'Budget'
   end
+
+  scenario "Email the event participants", :js => true do
+    sign_in_as_user(users(:tspmember))
+    visit events_path 
+    click_link "Death Star's destruction celebration"
+    
+    click_link "Email"
+    page.should have_content "Email the participants of Death Star's destruction celebration"
+    
+    # To check that users_for_event method is working properly
+    click_link "Select recipients"
+    find('a', :text => "All").click
+    page.should have_field('To', with: 'gial.ackbar@rebel-alliance.org,luke.skywalker@rebel-alliance.org,wedge.antilles@rebel-alliance.org,evram.lajaie@rebel-alliance.org,c3po@droids.com')
+    click_link "Select recipients"
+    find('a', :text => "Submitted").click
+    page.should have_field('To', with: 'wedge.antilles@rebel-alliance.org')
+    click_link "Select recipients"
+    find('a', :text => "Incomplete").click
+    page.should have_field('To', with: 'gial.ackbar@rebel-alliance.org,luke.skywalker@rebel-alliance.org,evram.lajaie@rebel-alliance.org,c3po@droids.com')
+    click_link "Select recipients"
+    find('a', :text => "Approved").click
+    page.should have_field('To', with: '')
+    click_link "Select recipients"
+    find('a', :text => "Accepted").click
+    page.should have_field('To', with: '')
+
+    click_link "Cancel"
+    current_path.should == event_path(events(:party))
+  end
```

Comment:
One thing I forgot... We should add a test that ensures that a user who has not the required permissions can not see the email link on the event page. And the same for users that are not logged in.